jupyter/Notebook Binding/Bar Tile XY.ipynb

```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table>\n",
+    "    <tr>\n",
+    "        <td><img src='../SystemLink_icon.png' /></td>\n",
+    "    </tr>\n",
+    "</table>\n",
+    "\n",
+    "## Bar Tile Binding Example\n",
+    "***\n",
+    "This example illustrates how a notebook execution can be bound to bar chart tile in a dashboard builder.\n",
+    "***\n",
+    "# Prerequisites\n",
+    "- None\n",
+    "\n",
+    "# Summary \n",
+    "This example illustrates how a notebook execution can be bound to bar tile in a dashboard builder. The result is set into the result object and then recorded using papermill to be consumed by the Notbook service. This example shows how a bar chart data can be modelled in the 'XY' format. In Index format, data is described as [[x0, x1, x2, ...]  [y0, y1, y2, ...]] format, where x0, x1, x2 etc shows the x-axis markings, and y0, y1, y2, ... shows the corresponding y-values.\n",
```

Comment:
This talks about the "XY" format, and then describes the specifics of the "Index" format. There's a separate notebook here with Index, should this description belong there instead?